apps/web/app/modules/Admin/AddCourse/AddCourse.tsx

```diff
@@ -20,16 +18,20 @@ import {
 } from "~/components/ui/select";
 
 import { useAddCourseForm } from "./hooks/useAddCourseForm";
+import Breadcrumb from "./components/Breadcrumb";
 
 const AddCourse = () => {
-  const navigate = useNavigate();
   const { form, onSubmit } = useAddCourseForm();
   const { data: categories } = useCategoriesSuspense();
   const [isUploading, setIsUploading] = useState(false);
   const { mutateAsync: uploadFile } = useUploadFile();
   const { isValid: isFormValid } = form.formState;
   const watchedImageUrl = form.watch("imageUrl");
   const imageUrl = form.getValues("imageUrl");
+  const maxDescriptionFieldLength = 800;
```

Comment:
**suggestion**: Maybe we can rename this const to `MAX_DESCRIPTION_FIELD_LENGTH`